File: src/network-controller/macCmdHandlers/ADRParamSetupAns.js
const BluebirdPromise = require('bluebird');

module.exports = function (devAddr, status) {
  let _this = this;
  return new BluebirdPromise((resolve, reject) => {
    _this.log.debug({
      label: 'MAC Command Ans',
      message: {
        ADRParamSetupAns: {
          Status: 'The payload is null',
        },
      },
    });

    resolve();
  });
};

File: src/network-controller/macCmdHandlers/ADRParamSetupAns.lua
local logger = require("../../log.lua")

return function(devAddr, status)
  logger.info(
    {
      label = "MAC Command Ans",
      message = {
        ADRParamSetupAns = {
          Status = "The payload is null"
        }
      }
    }
  )
end
